packages/terra-avatar/src/Avatar.scss

```diff
@@ -47,4 +48,8 @@
   .avatar-text-small {
     font-size: var(--terra-avatar-font-size-small, 1.143rem);
   }
+
+  .extra-white-space-fix {
```

Comment:
Where is this class used? I'd expect a markup update in this PR

Reply:
Ah that isn't needed, removed in this commit: https://github.com/cerner/terra-core/pull/1639/commits/f536b389d54312dfc16760e13d1cacb20093e6d9